netutils/ip.py

```diff
@@ -453,6 +453,29 @@ def cidr_to_netmaskv6(cidr: int) -> str:
     raise ValueError("Parameter must be an integer between 0 and 128.")
 
 
+def cidr_to_netwildmask(cidr: int) -> str:
```

Comment:
```suggestion
def cidr_to_wildcardmask(cidr: int) -> str:
```

Reply:
Function name changed to ``cidr_to_wildcardmask``. 
Added unit tests. 
Updated jinja convenience function list